kolibri/core/auth/api.py

```diff
@@ -438,7 +451,12 @@ def consolidate(self, items, queryset):
                     roles.append(role)
             item["roles"] = roles
             output.append(item)
-        output = sorted(output, key=lambda x: x[self.order_by_field])
+            if ordering_param.startswith("-"):
+                self.order_by_field = ordering_param[1:]
```

Comment:
Rather than reassign the property `order_by_field` here, I think best to treat that as immutable and juts reassign `ordering_param`, same in the else statement.